src/main/java/net/conjur/api/Variables.java

```diff
@@ -1,17 +1,19 @@
 package net.conjur.api;
 
+import javax.net.ssl.SSLContext;
+
 import net.conjur.api.clients.ResourceClient;
 
 public class Variables {
 
     private ResourceClient resourceClient;
 
-    public Variables(Credentials credentials) {
-        resourceClient = new ResourceClient(credentials, Endpoints.fromCredentials(credentials));
+    public Variables(Credentials credentials, SSLContext sslContext) {
+        resourceClient = new ResourceClient(credentials, Endpoints.fromCredentials(credentials), sslContext);
     }
 
-    public Variables(Token token) {
-        resourceClient = new ResourceClient(token, Endpoints.fromSystemProperties());
+    public Variables(Token token, SSLContext sslContext) {
+        resourceClient = new ResourceClient(token, Endpoints.fromSystemProperties(), sslContext);
     }
```

Comment:
I think we still need the bare constructors (without sslContext) for backward compatibility.

Reply:
Fixed.